Dockerfile

```diff
@@ -1,13 +1,28 @@
-# Using OpenJDK 8
-FROM broadinstitute/gatk:gatkbase-2.3.0
+# stage 1 for constructing the GATK zip
+FROM broadinstitute/gatk:gatkbase-2.3.0 AS gradleBuild
+LABEL stage=gatkIntermediateBuildImage
 
-# Location of the unzipped gatk bundle files
-ARG ZIPPATH
+RUN ls .
+ADD . /gatk
+WORKDIR /gatk
+
+RUN export GRADLE_OPTS="-Xmx4048m -Dorg.gradle.daemon=false" && /gatk/gradlew clean collectBundleIntoDir shadowTestClassJar shadowTestJar -Drelease=$RELEASE
+RUN cp -r $( find /gatk/build -name "*bundle-files-collected" )/ /gatk/unzippedJar/
+RUN unzip -o -j $( find /gatk/unzippedJar -name "gatkPython*.zip" ) -d /gatk/unzippedJar/scripts
```

Comment:
comments about the test jar would be useful to explain that it's necessary plumbing for tests but not for the build itself